### PR TITLE
Distrito Federal, Brazil

### PR DESCRIPTION
**Pull request by astoff**

### CODE REVIEW

File: sources/br/df/statewide.json [hunk 1]
```diff
@@ -16,8 +16,6 @@
             "function": "regexp",
             "field": "end_comp",
             "pattern": "^(Plano Piloto|Gama|Taguatinga|Brazlândia|Sobradinho|Planaltina|Paranoá|Núcleo Bandeirante|Ceilândia|Guará|Cruzeiro|Samambaia|Santa Maria|São Sebastião|Recanto das Emas|Lago Sul|Riacho Fundo|Lago Norte|Candangolândia|Águas Claras|Riacho Fundo II|Sudoeste e Octogonal|Varjão|Park Way|SCIA|Sobradinho II|Jardim Botânico|Itapoã|SIA|Vicente Pires|Fercal)",
```
Comment (trescube) [inline sources/br/df/statewide.json]: extraneous comma at the end of this line

**Comment by astoff** [comment]
The OA error is probably related to https://github.com/openaddresses/pyesridump/issues/29. What's wrong with the Travis test?

**Comment by astoff** [comment]
Re-run this, please.

**Comment by astoff** [comment]
Let's see if this one goes through now, with the increased timeout.

Re-run this, please.

**Comment by trescube** [comment]
Re-run this please

**Comment by astoff** [comment]
Log is not very helpful. I am guessing there is a 12-hour timeout for each process, and this source is taking longer than that to download... The funny thing is that the corresponding online map at http://www.geoservicos.segeth.df.gov.br/ArcGIS/rest/services/Operational/LOTE/MapServer/?f=jsapi seems to be reasonably fast to load.

**Comment by trescube** [comment]
I get this error when trying to download using [pyesridump](https://github.com/openaddresses/pyesridump):

```
2017-02-09 13:44:22,987 - cli.esridump - INFO - Built 4915 requests using OID enumeration method
Traceback (most recent call last):
  File "/Users/stephenhess/git/trescube/pyesridump/esridump/bin/esri2geojson", line 11, in <module>
    load_entry_point('esridump', 'console_scripts', 'esri2geojson')()
  File "/Users/stephenhess/git/trescube/pyesridump/esridump/cli.py", line 100, in main
    feature = next(feature_iter)
  File "/Users/stephenhess/git/trescube/pyesridump/esridump/dumper.py", line 365, in __iter__
    raise EsriDownloadError("Could not connect to URL", e)
esridump.errors.EsriDownloadError: ('Could not connect to URL', ReadTimeout(ReadTimeoutError("HTTPConnectionPool(host='www.geoservicos.segeth.df.gov.br', port=80): Read timed out. (read timeout=30)",),))
```

**Comment by astoff** [comment]
Re-run this, please.

**Comment by astoff** [comment]
This dataset seems generally better than the current one, where it is impossible to split house numbers from street name. Thus, I think it's OK to replace sources (instead of keeping both in parallel).

**Comment by astoff** [comment]
For some reason the Travis check never completed.

Re-run this, please.

**Comment by astoff** [comment]
Re-run this, please.

**Comment by astoff** [comment]
Closing and reopening to trigger Travis check

**Pull request merged**